extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -423,7 +424,7 @@ private function processInTrackPicture() {
 							'list' => 'imageusage',
 							'iutitle' => "File:{$inserted_image['il_to']}",
 							'iulimit' => $imageUsageLimit,
-						 );
+						);
 
 						try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS134)